scripts/zones/Stellar_Fulcrum/mobs/Kamlanaut.lua

```diff
@@ -63,6 +63,7 @@ end
 
 entity.onMobDeath = function(mob, player, isKiller)
     player:addTitle(xi.title.DESTROYER_OF_ANTIQUITY)
+    mob:getBattlefield():win()
```

Comment:
I'm curious why this is needed. Does it not win otherwise?

Reply:
Something is weird with this branch, its bringing back in stuff that was previously removed. Will need to relook at this.

Reply:
Ah wait I fixed this. The battlefield is supposed to win the moment he dies rather than on despawn. This is just to make it happen a bit earlier.